web/styles/app_components.css

```diff
@@ -1087,6 +1088,10 @@ div.overlay {
     outline: none;
     color: var(--color-text-default);
 
+    .dropdown_widget_value {
+        max-height: 23px;
+    }
+
```

Comment:
Although this CSS property will be applied to widgets at other places also but I think it will not affect the UI. Because in most of the places height is `20px`, only when a user pill in rendered height was `27px` which was looking odd. So, using this necessary.

Reply:
We would eventually need to set the height same for all dropdowns I guess.

Reply:
I think it is better to look into this once we finalize the whether we want to show user pill or "{user_icon} user-name" in options. So, currently I have not done any changes.